frontend/src/components/supp-allowances/TransportationAllowance.vue

```diff
@@ -40,10 +40,10 @@
   <div v-for="(model, index) in models" :key="model.supplementaryApplicationId ? model.supplementaryApplicationId : model.id" @input="update(model)">
     <v-card class="my-10" :class="{ greyTop: readOnly(model), 'basic-card': !readOnly(model) }" :disabled="readOnly(model)">
       <v-row class="pa-7 pt-10">
-        <v-col cols="1">
+        <v-col cols="10" md="1">
```

Comment:
I think md="2" lg="1" would look better

![image](https://github.com/user-attachments/assets/c18a03a8-d9b8-4409-8b6f-eddaceaab72a)

Reply:
done

Reply:
I'll approve this PR once you push your new commit